emscripten.py

```diff
@@ -797,8 +791,12 @@ def include_asm_consts(pre, forwarded_json, metadata):
 
     pre_asm_const = ''
 
-    if proxy_function:
-      pre_asm_const += '  if (ENVIRONMENT_IS_PTHREAD) { ' + proxy_debug_print(call_type) + 'return ' + proxy_function + '(' + proxy_args + '); } \n'
+    if shared.Settings.USE_PTHREADS:
+      sync = call_type == 'sync_on_main_thread_'
+      async = call_type == 'async_on_main_thread_'
```

Comment:
It looks like `async` is a keyword in Python 3.5+ so probably better to pick a different var name.

Reply:
Fixing, thanks.